scene/3d/gpu_particles_collision_3d.cpp

```diff
@@ -851,8 +918,15 @@ real_t GPUParticlesAttractor3D::get_directionality() const {
 }
 
 void GPUParticlesAttractor3D::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("set_cull_mask", "mask"), &GPUParticlesAttractor3D::set_cull_mask);
-	ClassDB::bind_method(D_METHOD("get_cull_mask"), &GPUParticlesAttractor3D::get_cull_mask);
+	ClassDB::bind_method(D_METHOD("set_collision_layer", "layer"), &GPUParticlesAttractor3D::set_collision_layer);
+	ClassDB::bind_method(D_METHOD("get_collision_layer"), &GPUParticlesAttractor3D::get_collision_layer);
+
+#ifndef DISABLE_DEPRECATED
+
+	ClassDB::bind_compatibility_method(D_METHOD("set_cull_mask", "mask"), &GPUParticlesAttractor3D::set_cull_mask);
```

Comment:
You don't use `bind_compatibility_method` for this, just use the normal method

Reply:
Fixed in https://github.com/godotengine/godot/pull/91858/commits/ebe4fb1585e947d5a0f5f6ecfdf7e74e1ede41ea